docs/operator/installing.md

```diff
@@ -0,0 +1,98 @@
+# Installing SpiceDB Operator
+
+## Create a Kubernetes Cluster
+
+The installation guide assumes you already have a kubernetes cluster available. 
+
+If you want to run the operator locally, we recommend:
+
+- [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+- [kind](https://kind.sigs.k8s.io)
+- [minikube](https://minikube.sigs.k8s.io)
+
+## Install the Operator with `kubectl`
+
+Confirm you are connected to the correct cluster with `kubectl config current-context`, and then:
+
+```sh
+kubectl apply --server-side -k github.com/authzed/spicedb-operator/config
+```
+
+After a succesful rollout, you will see the spicedb-operator pod running in the `spicedb-operator` namespace.
+
+```sh
+kubectl -n spicedb-operator get pods
+```
+
+## Create a `SpiceDBCluster`
+
+:::warning
+The commands documented do not configure best practices for production deployments.
+For example, TLS is disabled and data is not persisted when SpiceDB is shut down.
+See the guide for [selecting a datastore].
+:::
+
+You can now create and configure SpiceDB clusters by writing `SpiceDBCluster` objects, for example:
```

Comment:
You can now create and configure a SpiceDB cluster by writing a `SpiceDBCluster` object. For example: